Cargo.toml

```diff
@@ -19,10 +19,11 @@ std = ["unsigned-varint/std", "tiny-multihash-derive/std"]
 multihash-impl = ["derive", "all"]
 derive = ["tiny-multihash-derive"]
 test = ["multihash-impl", "quickcheck", "rand"]
-all = ["blake2b", "blake2s", "sha1", "sha2", "sha3", "strobe"]
+all = ["blake2b", "blake2s", "blake3", "sha1", "sha2", "sha3", "strobe"]
 scale-codec = ["parity-scale-codec"]
 serde-codec = ["serde", "generic-array/serde"]
 
+blake3 = ["blake3_"]
```

Comment:
Introducing a new feature doesn't seem required

Reply:
@mriise I think you can just use `"blake3"` in the `all` feature. If you name an optional dependency in a feature (like `all`) it will be used. So we are lucky and you can just use `blake3` and you won't need any additional renaming of things or even a dedicated feature.

Reply:
ok!